lighthouse-core/audits/estimated-input-latency.js

```diff
@@ -45,23 +48,34 @@ class EstimatedInputLatency extends Audit {
       throw new LHError(LHError.errors.NO_FMP);
     }
 
-    const latencyPercentiles = TracingProcessor.getRiskToResponsiveness(tabTrace, startTime);
-    const ninetieth = latencyPercentiles.find(result => result.percentile === 0.9);
-    const rawValue = parseFloat(ninetieth.time.toFixed(1));
+    const events = TracingProcessor.getMainThreadTopLevelEvents(tabTrace, startTime);
+
+    const candidateStartEvts = events.filter(evt => evt.duration >= 10);
+
+    let worst90thPercentileLatency = 16;
+    for (const startEvt of candidateStartEvts) {
+      const latencyPercentiles = TracingProcessor.getRiskToResponsiveness(
+        events,
+        startEvt.start,
+        startEvt.start + ROLLING_WINDOW_SIZE
```

Comment:
if there's a last argument of `[0.9]` then no need for the `find()` below (since dropping the `extendedInfo`...though if we keep EIL, there's arguably usefulness in keeping other percentiles :)

Reply:
done